CHANGELOG.md

```diff
@@ -1,3 +1,61 @@
+# 0.0.109 - 2022-06-30
+
+## API Updates
+ * `ChannelManager::update_channel_config` has been added to allow the fields
+   in `ChannelConfig` to be changed in a given channel after open (#1527).
+ * If we reconnect to a peer which proves we have a stale channel state, rather
+   than force-closing we will instead panic to provide an opportunity to switch
+   to the latest state and continue operating without channel loss (#1564).
+ * A `NodeAlias` struct has been added which handles string sanitization for
+   node aliases via the `Display` trait (#1544).
+ * `ProbabilisticScoringParameters` now a `banned_nodes` set which we will
+    never route through when using a `ProbabilisticScorer` (#1550).
```

Comment:
```suggestion
    avoid during path finding (#1550).
```

Reply:
Mh, I mainly suggested that because its factually wrong: we would not refuse routing (as in 'forwarding') to/through such a node, but we would not include them *for our own payment paths*.

Reply:
Yeah, works for me!